### ISSUE TITLE
publication is not up to date

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/neuro.polymtl.ca#54]

**Comment by benjamindeleener** [comment]
My gsheet is [here](https://docs.google.com/spreadsheets/d/16-OXLtXX87C_Ae28a2AKCAOn0I3_qjnAfzi6whLb8HE/edit?usp=sharing). Not filled yet.

**Comment by namgo** [comment]
@benjamindeleener (and others), if you can let me know when you have filled in your sheets, I'll be able to add them to the publications on the website now.

**Comment by jcohenadad** [comment]
@namgo I think you can just create the gsheets for the other PIs so we can move things forward-- thanks!

**Comment by namgo** [comment]
@jcohenadad I think I wasn't sure how to proceed and then totally dropped this without realizing I was doing so!

If I understand correctly all I need to do is create gsheets with the template presented through your gsheet?

**Comment by namgo** [comment]
https://docs.google.com/spreadsheets/d/1IohadXl6Z9GTYQGx-sOUKT-fLGPuGSkuY8OgYtPoU3w/edit?usp=sharing Eva Alonso Ortiz
https://docs.google.com/spreadsheets/d/16-OXLtXX87C_Ae28a2AKCAOn0I3_qjnAfzi6whLb8HE/edit?usp=sharing (Benjamin DeLeener's pre-existing sheet)
https://docs.google.com/spreadsheets/d/1dEUBYf17hNM22dqV4zx1gsh3Q-d97STnRB4q7p9nQ54/edit#gid=742520500 (Nikola pre-existing sheet)


@jcohenadad I just copied Benjamin's sheet to create Nikola and Eva's, which I think I did correctly? They reside in the root folder of neuropoly's shared folder at the moment.

I could probably loop the bibeasy import across the different sheets, but duplication would be a challenge. Provided the formatting follows Benjamin and yours, I think it should be doable.

**Comment by jcohenadad** [comment]
Amazing! Thank you. Yes, we need to deal with duplication. One suggestion would be to look for matched name + title + journal. If these three match, then skip the output from bibeasy

**Comment by jcohenadad** [comment]
> https://docs.google.com/spreadsheets/d/1dEUBYf17hNM22dqV4zx1gsh3Q-d97STnRB4q7p9nQ54/edit#gid=742520500 (Nikola pre-existing sheet)

no, that's my gsheet-- Nikola needs to have its separate gsheet URL

Eva's one looks empty

Benjamin's too

**Comment by namgo** [comment]
I misunderstood then, I thought I just needed to make a gsheet format for PI's to fill in and then add support for the importing of their citations.

**Comment by jcohenadad** [comment]
> Should I be importing their citations from CCV, or am I missing something?

no it's fine, i'll do it-- unless @evaalonsoortiz @benjamindeleener you do it-- thx

**Comment by namgo** [comment]
Thanks! I believe I created another (empty, but templated) sheet for Nikola that is currently in the root folder where the Eva's is if that helps.
